packages/core/src/app/ui/modal/ModalTrigger.tsx

```diff
@@ -49,7 +49,13 @@ export default class ModalTrigger extends Component<ModalTriggerProps, ModalTrig
     }
 
     private handleOpen: () => void = () => {
-        document.body.scrollIntoView();
+        const wordpressCheckout = parent.document.getElementById('bc-embedded-checkout');
```

Comment:
`bc-embedded-checkout` is the name of the embedded checkout container ([sample embedded checkout in Wordpress](https://bcsandbox.jabctools.com/checkout/)).

Reply:
Do you think move this to a const outside? Since if we do change the container id will be easier to track.